modules/installation-configuration-parameters.adoc

```diff
@@ -1000,7 +1000,11 @@ Additional {rh-virtualization} configuration parameters for machine pools are de
 |Number
 
 |`<machine-pool>.platform.ovirt.vmType`
-|Optional. The VM workload type, such as `high-performance`, `server`, or `desktop`.
+|Optional. The VM workload type, such as `high-performance`, `server`, or `desktop`.  By default, master nodes use `high-performance`, and worker nodes use `server`. For details, see link:https://access.redhat.com/documentation/en-us/red_hat_virtualization/4.4/html-single/virtual_machine_management_guide/index#Virtual_Machine_General_settings_explained[Explanation of Settings in the New Virtual Machine and Edit Virtual Machine Windows] and link:https://access.redhat.com/documentation/en-us/red_hat_virtualization/4.4/html-single/virtual_machine_management_guide/index#Configuring_High_Performance_Virtual_Machines_Templates_and_Pools[Configuring High Performance Virtual Machines, Templates, and Pools] in the _Virtual Machine Management Guide_.
```

Comment:
nitpicky, so disregard if you are so inclined...maybe remove the comma from "...`high-performance`, and worker...".  It doesn't seem like there's a need for a mental pause.

(I was going to suggest prefixing the first link text with "Appendix: Explanation of...", but that prolly reqs organization-wide agreement.  I was a little surprised to find myself in "A1. Explanation..." and that's what made me think of it.)

Reply:
@mikemckiernan 

> nitpicky, so disregard if you are so inclined...maybe remove the comma from "...`high-performance`, and worker...". It doesn't seem like there's a need for a mental pause.

Eh, I think you're right that it's not necessary grammatically, but in this case, at least for me, the mental pause helps to digest the info. Maybe someone more familiar with the concept doesn't need it, but I'm inclined to leave it.

> (I was going to suggest prefixing the first link text with "Appendix: Explanation of...", but that prolly reqs organization-wide agreement. I was a little surprised to find myself in "A1. Explanation..." and that's what made me think of it.)
I prefer to leave it as generic as possible. For all I know, tomorrow someone might decide that this is a chapter and not an appendix.